test/unit/helpers/bson.js

```diff
@@ -4,20 +4,17 @@ var chai = require('chai');
 var expect = require('chai').expect;
 var bson = require('../../../helpers/bson');
 
-/**
- * Very simple test for the bson wrapper in the /helpers folder.
- */
 describe('bson', function () {
 
 	describe('quick check', function () {
 
 		it('pass serializes and deserializes an object', function () {
+			var originalObject = { num: 200,  desc: 'silly things.'};
```

Comment:
That object is too simple, missing some stuff like properties with `null`/`undefined` values.

Reply:
I have removed the test in order to focus on bignum passing through the team.